server/src/dbUtils.ts

```diff
@@ -15,6 +17,17 @@ function getStringOrUndefined(value: unknown): string | undefined {
 	throw `Expected value to be string or null, got: ${typeof value}`;
 }
 
+function getTagsOrUndefined(value: unknown): string[] | undefined {
```

Comment:
I'd name this 'getStringArrayOrUndefined', cause we had only abstract functions here so far :) not related to our 'business' logic, but to types

Reply:
I have named it so, because inside the function there is isTags check, that checks got tags. Maybe then I need to change the way this function works as well. Could you have a 2nd look and lemme know what u think :)